package.json

```diff
@@ -67,7 +67,7 @@
     "broccoli-funnel": "^2.0.2",
     "ember-cli-babel": "^7.13.2",
     "ember-inflector": "^3.0.1",
-    "eslint-plugin-node": "^10.0.0",
+    "eslint-plugin-node": "^11.0.0",
```

Comment:
@runspired did we intend for this to be a devdep?  there's no reason for it to be a dep is there?